src/test/java/org/minions/devfund/cristhian/WhackAMoleTest.java

```diff
@@ -0,0 +1,174 @@
+package org.minions.devfund.cristhian;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * WhackAMole test.
+ */
+
+public class WhackAMoleTest {
+    private static final int ONE = 1;
+    private static final int THREE = 3;
+    private static final int FIVE = 5;
+    private static final int TEN = 10;
+    private static final String GRID = "*****";
+
+    /**
+     *
+     */
+    @Test
+    public void testPrintGridMethod() {
+        WhackAMole myWhackAMole = new WhackAMole(FIVE, FIVE);
+        String expectedResult = GRID + "\n"
+                + GRID + "\n"
+                + GRID + "\n"
+                + GRID + "\n"
+                + GRID + "\n";
+        assertEquals(expectedResult, myWhackAMole.printGrid());
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testPlaceMethod() {
+        WhackAMole myWhackAMole = new WhackAMole(FIVE, FIVE);
+        myWhackAMole.place(THREE, THREE);
+        myWhackAMole.place(THREE, THREE);
+        String expectedResult = GRID + "\n"
+                + GRID + "\n"
+                + GRID + "\n"
+                + "***M*" + "\n"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "***M*" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)